.travis.yml

```diff
@@ -61,9 +65,17 @@ jobs:
       before_script:
         - composer install
         - composer update
+    - <<: *phpunit
+      php: 7.3
+      env: PHPUnit=latest
+      before_script:
+        - composer install
+        - composer update
```

Comment:
I don't understand why `install` and `update` is used here. `composer update` would be enough imho.

Reply:
Yes , this is true. Was just copying it over. Did it. Can you check again?

Reply:
I did this to log in the ci de difference between the locked dependencies and what is new.

Reply:
Too smart. So you think we should still keep it?

Reply:
If composer.lock stays it gives you an overview what would change. If it's dropped it can be just install.

Reply:
@t3easy Good point. For the moment composer.lock stays, but i´ll try to get rid of it soon. So we keep it for the moment.

Reply:
@t3easy @simonschaufi Could you both try to approve the PR?